File: frontend/src/types/meeting.ts
export interface Meeting {
  id: number;

  title: string;

  description: string;

  organizer: string;

  meetingDate: string;

  startTime: string;

  endTime: string;

  location: string;

  transcript?: string;

  audioFile?: string;

  status: "Scheduled" | "Completed" | "Cancelled";

  createdAt: string;

  updatedAt: string;
}

File: backend/api-service/app/models/meeting.py
from sqlalchemy import Column, Integer, String, Text
from app.models.base import Base


class Meeting(Base):
    __tablename__ = "meetings"

    id = Column(Integer, primary_key=True, index=True)

    title = Column(String(200), nullable=False)

    description = Column(Text)

    meeting_date = Column(String(50))

    start_time = Column(String(20))

    end_time = Column(String(20))

    organizer = Column(String(100))

    join_url = Column(String(255))